sys/parsoid.js

```diff
@@ -157,7 +157,7 @@ class ParsoidProxy {
             res.headers = res.headers || {};
             res.headers['x-parsoid-variant'] = variant;
             return P.resolve(res);
-        }).catch({ status: 404 }, (e) => {
+        }).catch({ status: 404 }, { status: 421 }, (e) => {
```

Comment:
Should we log a warning in case of 421? that way some critical problem woundln go unnoticed.

Reply:
@Pchelolo {{done}}